t/t0402-block-command-on-gvfs.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/sh
+
+test_description='block commands in GVFS repo'
+
+. ./test-lib.sh
+
+test_expect_success 'test fsck' '
+	git config alias.kcsf fsck &&
```

Comment:
Please use `test_config`. That has the advantage of unsetting the config setting after the test case is done.